versioneer/tag_list.go

```diff
@@ -59,8 +59,11 @@ func (tl TagList) Images() TagList {
 
 	newTags := []string{}
 	for _, t := range tl.Tags {
-		// We have to filter "-img" tags outside the regexp because golang regexp doesn't support negative lookaheads.
-		if regexpObject.MatchString(t) && !strings.HasSuffix(t, "-img") {
+		// Golang regexp doesn't support negative lookaheads so we filter some images
+		// outside regexp. Here we filter out `-img` and `-uki` artifacts.
+		if regexpObject.MatchString(t) &&
+			!strings.HasSuffix(t, "-img") &&
```

Comment:
maybe time to extract this to a list?

Reply:
Good point! Done here: https://github.com/kairos-io/kairos-sdk/pull/262/commits/eafc701ad79fcd49d1f0b59f5b27894985a5227f